app/views/admin/workshops/_invitation_management.html.haml

```diff
@@ -25,15 +25,15 @@
     .medium-6.columns
       %h4
         Students
-        %label.label.round.secondary #{@attending_students.count}#{(" of " + @workshop.host.seats.to_s) if @workshop.has_host?}
+        %label.label.round.secondary #{@attending_students.count} of #{@workshop.student_spaces}
```

Comment:
I'm not sure why we need a variable for `@attending_students`. Investigate and remove.

Reply:
This is a decorated collection and used elsewhere, so I'd rather not change it on this PR.